requirements.txt

```diff
@@ -1,30 +1,2 @@
-Django>=1.9,<1.10
-cookiecutter==1.0.0
-psycopg2cffi
-raven==5.0.0
-markdown
-django-libsass
-certifi
-dj-database-url
-django-cas-ng==3.5.4
-django-mptt==0.8.5
-requests
-elastic-git
-unicore.content
-wagtail==1.9
-responses
-babel
-wagtailmedia==0.1.3
-django-extensions
-celery<4.0
-redis
-six>=1.9
-django-google-analytics-app==2.1.4
-mote-prk==0.2.1
-beautifulsoup4
-ujson==1.35
-djangorestframework==3.3.2
-cached-property
-setuptools==18.5
-molo.profiles<6.0.0,>=5.0.0
-django-el-pagination==3.1.0
+# Our dependencies are all specified in setup.py
```

Comment:
@JayH5 we moved away from this approach because it requires any changes to version for someone to make a change to `setup.py`. It's a similar approach for why the version is removed from `setup.py` into its own file.

Is there any reason why you prefer moving everything to setup.py?